Makefile

```diff
@@ -481,7 +481,7 @@ endif
 	rm -fr $(BUILDROOT)/julia-$(JULIA_COMMIT)
 
 # this target does not accept BUILDROOT
-light-source-dist.tmp: $(JULIAHOME)/doc/_build/html
+light-source-dist.tmp: $(BUILDROOT)/doc/_build/html/en/index.html
```

Comment:
the docs should now be able to handle an out-of-tree build (and that's what this target is defined in terms of, elsewhere in the file), but other pieces of this target most likely won't